modules/proxyconnection/CustomSignalingLayer.js

```diff
@@ -108,12 +108,6 @@ export default class CustomSignalingLayer extends SignalingLayer {
         return false;
     }
 
-    /**
-     * @inheritDoc
-     */
-    setTrackSourceName(ssrc, sourceName) { // eslint-disable-line no-unused-vars
-    }
-
```

Comment:
The implementation of `setSSRCOwner` and `updateSsrcOwnersOnLeave` here and in `SignalingLayerImpl` have diverged. Since Spots's wireless connection uses a mock signaling connection, I think we can remove the impl of those methods here but it needs to be tested.

Reply:
@saghul WDYT?

Reply:
Agreed! I think it's safe, but we can test against alpha, once it lands.

Reply:
I don't see how anything would break but testing once it lands is always a good idea :)